README.md

```diff
@@ -1,4 +1,11 @@
-# The Next Version of C3 Web
+# The Next Version of C3 Web 
+## Status 
+[![Build Status](https://travis-ci.org/C3Junior/c3-web-next.svg?branch=master)](https://travis-ci.org/C3Junior/c3-web-next)
+[![License](https://img.shields.io/badge/licence-GNU-blue.svg)](http://www.gnu.org/licenses/gpl-3.0.en.html)
```

Comment:
did we agree on changing the license? :)
Original C3 was licensed under New BSD license, see https://en.wikipedia.org/wiki/BSD_licenses#3-clause_license_.28.22Revised_BSD_License.22.2C_.22New_BSD_License.22.2C_or_.22Modified_BSD_License.22.29
GPL 3.0 is much less permissive.